app/middlewares/errors.js

```diff
@@ -12,7 +12,8 @@ const statusCodes = {
   [errors.ENCRYPTION_ERROR]: 500,
   [errors.INTERNAL_ERROR]: 500,
   [errors.NOT_FOUND_ERROR]: 404,
-  [errors.AUTHENTICATION_ERROR]: 401
+  [errors.AUTHENTICATION_ERROR]: 401,
+  [errors.AUTHORIZATION_ERROR]: 401
```

Comment:
Authorization error should be 403 code

Reply:
ok, I was confused with this because in the wolox documentation 401 is called unauthorized and 403 is forbidden

Reply:
401 is normally used for user identification, e.g. bad credentials or session token. 403 is when you can identify the client, but the action is not allowed. In this case, `forbidden error` and `authorization error` should be the same.